### ISSUE TITLE
Improve Documentation

### ISSUE THREAD

**Issue by iusehooks** [iusehooks/redhooks#1]
We wish to improve the documentation. Any help is welcome!

**Comment by jyothisankarreddy** [comment]
I can help for this.

**Comment by iusehooks** [comment]
Thanks :)

**Comment by gazing3668** [comment]
Do you still want help? I'm willing.

**Comment by iusehooks** [comment]
Yes Thanks. The idea is to create a proper docs with https://docusaurus.io/ .

**Comment by stardust3dd** [comment]
Do you still want help? I'm willing.

**Comment by gazing3668** [comment]
Would you mind telling me a little bit more? I'm completely new to this, sorry.

**Comment by iusehooks** [comment]
Taking inspiration by the contents in the actual readme.md, you can create a static website using https://docusaurus.io/docs/en/installation .  The examples one their website should guide you to get started with it.

**Comment by lawrence-dass** [comment]
I'd like to work on this, please.

**Comment by Ayan4** [comment]
Hey, I'd like to help if the issue is still open.
